src/main/java/com/example/NiRi/modules/LoginRequest.java

```diff
@@ -0,0 +1,30 @@
+/*package com.example.NiRi.modules;
+
+public class LoginRequest {
+     private String email;
+     private String password;
+
+     public LoginRequest(String email, String password) {
+
+         this.email = email;
+         this.password = password;
+
+     }
+
+     public String getEmail() {
+            return email;
+        }
+
+     public void setEmail(String email) {
+            this.email = email;
+        }
+
+     public String getPassword() {
+            return password;
+        }
+
+     public void setPassword(String password) {
+            this.password = password;
+        }
+
+}*/
```

Comment:
do we still need this?

Reply:
I left it as it is to add authentication sometime later after all the modules are completed